File: go/docmost/models/page_history.go
package models

// PageHistory from OpenAPI.
type PageHistory struct {
	Id *string `json:"id,omitempty"`
	PageId *string `json:"pageId,omitempty"`
	Title *string `json:"title,omitempty"`
	Icon *string `json:"icon,omitempty"`
	SlugId *string `json:"slugId,omitempty"`
	Slug *string `json:"slug,omitempty"`
	Content map[string]any `json:"content,omitempty"`
	Version *string `json:"version,omitempty"`
	LastUpdatedById *string `json:"lastUpdatedById,omitempty"`
	ContributorIds *string `json:"contributorIds,omitempty"`
	SpaceId *string `json:"spaceId,omitempty"`
	WorkspaceId *string `json:"workspaceId,omitempty"`
	CreatedAt *string `json:"createdAt,omitempty"`
	UpdatedAt *string `json:"updatedAt,omitempty"`
}

File: c#/src/Docmost.Sdk/Models/PageHistory.cs
namespace Docmost.Sdk.Models;

public class PageHistory
{
    [System.Text.Json.Serialization.JsonPropertyName("id")]
    public string? Id { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("pageId")]
    public string? PageId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("title")]
    public string? Title { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("icon")]
    public string? Icon { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("slugId")]
    public string? SlugId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("slug")]
    public string? Slug { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("content")]
    public System.Text.Json.JsonElement? Content { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("version")]
    public string? Version { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("lastUpdatedById")]
    public string? LastUpdatedById { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("contributorIds")]
    public string? ContributorIds { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("spaceId")]
    public string? SpaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("workspaceId")]
    public string? WorkspaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("createdAt")]
    public string? CreatedAt { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("updatedAt")]
    public string? UpdatedAt { get; set; }

}
